README.md

```diff
@@ -2,15 +2,17 @@
 #### Encode or decode base64 or base32 using a custom alphabet
 
 ```markdown
-usage: custombase.py [-h] -in STRING_IN [-c CUSTOMALPHABET] [-32] [-hex]
+usage: custombase.py [-h] -in STRING_IN -f | FILE_IN [-c CUSTOMALPHABET] [-32] [-hex]
```

Comment:
I think  if the | was before -f it would be more clear to the user that one or the other is needed